sdk/nodejs/cmd/dynamic-provider/index.ts

```diff
@@ -333,7 +424,12 @@ function resultIncludingProvider(result: any, props: any): any {
 // rejected the resource, or an initialization error, where the API server has accepted the
 // resource, but it failed to initialize (e.g., the app code is continually crashing and the
 // resource has failed to become alive).
```

Comment:
If we wanted to  `/**` / `*/` this while we were here I would not object :smile_cat: